packages/design-system/src/styles/_reset.scss

```diff
@@ -0,0 +1,344 @@
+/* Core root defaults */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+// Remove all animations, transitions and smooth scroll
+// for people that prefer not to see them
+@media (prefers-reduced-motion: reduce) {
+  html:focus-within {
+    scroll-behavior: auto;
+  }
+
+  *,
+  *::before,
+  *::after {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    scroll-behavior: auto !important;
+    transition-duration: 0.01ms !important;
+  }
+}
+
+/* Core body defaults */
+
+body {
+  background-color: var(--color-background);
+  color: var(--color-base);
+  font-family: var(--font-body);
+  font-size: var(--font-size-base);
+
+  // 👀 CMS-WDS/scout uses these reset styles
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  line-height: var(--font-line-height-base);
+  margin: 0;
+  min-height: 100vh;
+  text-rendering: optimizeSpeed;
+}
+
+// 👀 CMS-WDS/sep-screener uses this reset style
+// I've seen similar styles in other repos and I'm curious what it's solving
+#main[tabindex='-1']:focus,
+h1[tabindex='-1']:focus {
+  outline: 0;
+}
+*[tabindex='-1']:focus {
+  outline: 0;
+}
```

Comment:
I imagine this is getting rid of the focus outline for non-focusable elements that have explicitly added the `tabindex="-1"` in order to make them focusable. I'd love @SparrowSpit's thoughts on this. Should we by default hide focus rings for such elements?

It also looks like the `#main` and `h1` are redundant with the `*`, and I'm curious if we even need `*` or if the selector could be just `[tabindex='-1']:focus`

Reply:
Yeah, it's common to not display focus rings for non-interactive elements like when activating a skip link will typically send focus to a <main> or a heading element. The caveat to this, of course, would be if the heading element also happens to be a link.